css-fonts/styles.css

```diff
@@ -0,0 +1,5 @@
+body{
+  font-family: 'Abel', sans-serif;
+  font-family: 'Architects Daughter', cursive;
```

Comment:
If you set the same property twice in one rule, the second one is used, and the first is discarded. You need to apply these two font families to different elements in different rules